src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -672,4 +837,168 @@ public static String replaceCodes(String text, char from) {
     public static String replaceCodes(String text, char from, char to) {
         return factory.replaceLegacyCodes(text, from, to);
     }
+
+    /**
+     * Helper class that allows easier/chained calls to the format method
+     * without extra steps
+     */
+    public static final class FormatBuilder implements ResettableBuilder<FormatBuilder> {
```

Comment:
It seems to me that this ties the 'context' of a template to the template itself, is that a good thing? Conceptually, I feel like a template is completely separate from its arguments, and it just takes them in order to output something new.